apps/api/.env.example

```diff
@@ -48,16 +50,19 @@ SMTP_USER=
 SMTP_PASSWORD=
 
 # Mnemonic for master funding & signing account
-FUNDING_MNEMONIC=
+
+FUNDING_MNEMONIC= ;from the sandbox account you created (check the README.md)
 
 # Used for passphrase encryption
-SECRET=
+
+SECRET= ;any string
```

Comment:
Should we recommend a min length or maybe something obfuscated? This one seems important for something like that.

Reply:
One thing we frequently use to generate keys on the fly is:
```sh
$ openssl rand -hex 32 | pbcopy
```

Which will generate something like: `5f6a12942bb2c44eb70cf1b71ef1c211cc2d966981f76e6315bd086fe54cac16`. Could be worth mentioning at the top of the .env file — or not. Just a good thing to be aware of!

Reply:
Oh that's awesome, I'll definitely add that in!